build.gradle

```
@@ -25,29 +25,43 @@ group 'bio.terra'
 version '0.0.16-SNAPSHOT'
 sourceCompatibility = JavaVersion.VERSION_1_8
 
+def OPENCENSUS_VERSION = '0.28.2'
+
 dependencies {
     implementation group: 'bio.terra', name: 'stairway', version: '0.0.37-SNAPSHOT'
+    implementation group: 'ch.qos.logback.contrib', name: 'logback-json-classic', version: '0.1.5'
+    implementation group: 'ch.qos.logback.contrib', name: 'logback-jackson', version: '0.1.5'
+    implementation group: 'com.google.cloud', name: 'google-cloud-core', version: '1.94.0'
+    implementation group: 'com.google.apis', name: 'google-api-services-logging', version: 'v2-rev662-1.18.0-rc'
     implementation group: 'io.kubernetes', name: 'client-java', version: '10.0.0'
-    implementation group: "io.opencensus", name: "opencensus-api", version: "0.28.2"
+    implementation group: 'io.opencensus', name: 'opencensus-api', version: OPENCENSUS_VERSION
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.11'
-    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.28'
-    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
+    implementation group: 'org.hashids', name: 'hashids', version: '1.0.3'
     implementation group: 'org.liquibase' , name: 'liquibase-core', version: '3.10.0'
-    implementation group: 'org.springframework' , name: 'spring-context', version: '5.3.2'
+    implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.28'
+    implementation group: 'org.springframework' , name: 'spring-context', version: '5.2.7.RELEASE'
+    implementation group: 'org.springframework' , name: 'spring-core', version: '5.2.7.RELEASE'
```

versions controlled by spring:
```suggestion
    implementation group: 'org.springframework' , name: 'spring-context'
    implementation group: 'org.springframework' , name: 'spring-core'
```